models/classesDao.js

```diff
@@ -0,0 +1,233 @@
+    c.host_id = 34 AND 
+    c.deleted_at IS NULL
+  GROUP BY 
+    c.id, c.title, c.summary, c.price, tc.name, sc.name, p.address
+  ORDER BY 
+    c.title;
+  `;
+  return await appDataSource.query(query,[hostId]);
+};
+const findTopCategoryIdByName = async (topCategoryName,transaction) => {
+  const query = `SELECT id FROM top_categories WHERE name = ?`;
+  return await transaction.query(query, [topCategoryName]);
+};
+const findSubCategoryIdByName = async (subCategoryName, topCategoryId,transaction) => {
+  const query = `SELECT id FROM sub_categories WHERE name = ? AND top_category_id = ?`;
+  return await transaction.query(query, [subCategoryName, topCategoryId]);
+};
+const createPlace = async (address, latitude, longitude, transaction) => {
+  const query = `INSERT INTO places (address, latitude, longitude, created_at) VALUES (?, ?, ?, NOW())`;
+  const placeResult = await transaction.query(query, [address, latitude, longitude]);
+  return placeResult.insertId;
+};
+const createClass = async(host_id, title, summary, content, price, placeId, getTopCategoryId, getSubCategoryId, transaction) => {
+  const query = `INSERT INTO classes (host_id, title, summary, content, price, sales, place_id, top_category_id, sub_category_id, created_at) VALUES (?, ?, ?, ?, ?, 0, ?, ?, ?, NOW())`;
+  const classResult = await transaction.query(query, [host_id, title, summary, content, price, placeId, getTopCategoryId, getSubCategoryId]);
+  return classResult.insertId;
+};
+const createImage = async (classId, name, imageSource, transaction) => {
+  const query = `INSERT INTO images (class_id, name, image_source) VALUES (?, ?, ?)`;
+  const imageResult = await transaction.query(query, [classId, name, imageSource]);
+  return imageResult;
+};
+const schedulesStatusCheckByclassId = async(classId) => {
+  const query = `SELECT * FROM schedules WHERE class_id = ? AND (status = 1 AND enrolled_member != 0)`;
+  return await appDataSource.query(query,[classId]);
+};
+const deleteClassByClassId = async(classId) => {
+  const query = `UPDATE classes SET deleted_at = CURRENT_TIMESTAMP WHERE id = ?;`
+  return result = await appDataSource.query(query,[classId]);
+};
+const getPlaceIdByClassId = async(classId,transaction) => {
+  const query = `SELECT place_id FROM classes WHERE id= ?`
+  return await transaction.query(query,[classId]);
+};
+//업데이트
+const modifyPlaces = async(getPlaceId,address,latitude,longitude,transaction) => {
+  const query = `UPDATE places SET address = ?, latitude = ?, longitude = ? WHERE id = ?`
+  return await transaction.query(query,[address,latitude,longitude,getPlaceId])
+};
+const modifyClasses = async(classId,title,summary,content,price,getTopCategoryId,getSubCategoryId,transaction) => {
+  const query = `UPDATE classes SET title = ?, summary = ?, content = ?, price = ? , top_category_id = ?, sub_category_id = ? WHERE id = ?`
+  return await transaction.query(query,[title,summary,content,price,getTopCategoryId,getSubCategoryId,classId])
```

Comment:
```suggestion
const modifyClasses = async(classId,title,summary,content,price,getTopCategoryId,getSubCategoryId,transaction) => {
  const query = `
    UPDATE classes 
    SET 
      title = ?, 
      summary = ?, 
      content = ?, 
      price = ? , 
      top_category_id = ?, 
      sub_category_id = ? 
    WHERE id = ?`
  return await transaction.query(query,
    [title, summary, content, price, getTopCategoryId, getSubCategoryId, classId]
  )
```
진서님 다 좋은데 가독성 강화가 좀 많이 필요할 것 같아요